src/contexts/IConfigProvider.ts

```diff
@@ -10,10 +10,16 @@ export interface Config {
     [key in Practices]?: PracticeImpact;
   };
   tokens?: {
-    [key in Service]: string;
+    [key in Service]: string | EslintConfig;
```

Comment:
@adelkahomolova  `EslintConfig `??? Why the key `token` has an interface of `Service: ESLintConfig` (`"Slack": { ... }`)?

Reply:
It seems it should be on line 10 probably?

Reply:
`getOverridenPractice` should have a return type, it's maybe the pain point